shared/src/test/diff/mlscript/BadMethods.mls

```diff
@@ -753,6 +753,9 @@ class Test7C: Test7A & Test7B
 //│ ╙──       	         ^^^^^^^^^^
 //│ Defined class Test7C
 
+Test7C.Mth7A
+//│ res: error
```

Comment:
While I agree that the logic has room for improvement, I don't think an error should be reported here. It would just be identical to the one reported at the definition. There's nothing more to report here until the definition is fixed.

Reply:
There are no errors reported here. And it behaves the same as before my changes AFAIK. I just wanted to exercise this specific case in the tests.

Reply:
Oh I thought it was related to the note above `MethodType`, which says that the error should be checked at the call site.

Reply:
No, in that note I meant the call sites of the `toPT`/`bodyPT` methods.